src/test/java/com/galvanize/customer/controllers/CustomerRestControllerTests.java

```diff
@@ -27,47 +33,59 @@
 
     @Autowired
     CustomerRepository repository;
-
+    String jsonPath ;
     @Before
-    public void setUp() throws Exception {
+    public void setUp() {
 
+        //Set up
+        jsonPath = "{\"address\":\"55 Hawthorne Rd\",\"city\":\"Raleigh\",\"dateJoined\":\"12/12/2019\",\"firstName\":\"Eva\",\"lastName" +
+                "\":\"Mendez\"," +
+                "\"phoneNumber\":\"9673019375\",\"state\":\"NC\",\"zip\":\"27603\"}";
     }
 
     @Test
     public void addCustomer() throws Exception {
+        //Execute and Assertion
+        mvc.perform(post("").contentType(MediaType.APPLICATION_JSON)
+        .content(jsonPath))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("customerId").exists());
     }
 
     @Test
     public void getCustomersByState() throws Exception {
+        mvc.perform(get("/State/NC"))
+                .andExpect(status().isOk());
     }
 
     @Test
     public void getAllCustomers() throws Exception {
+        mvc.perform(get("/"))
+                .andExpect(status().isOk());
     }
 
-    @Test
+   @Test
     public void getCustomerById() throws Exception {
+        mvc.perform(get("/id/5"))
+                .andExpect(status().isOk());
     }
 
     @Test
     public void updateCustomerPhoneNumber() throws Exception {
+        Optional<Customer> customerList = service.getCustomerById(3L);
```

Comment:
Looks like you are relying on data existing in the db.  You should create this data first, then test for it.  Remember - keep all tests isolated.